spanner/tests/system.py

```diff
@@ -531,6 +536,60 @@ def test_read_w_manual_consume(self):
         self.assertEqual(streamed._current_row, [])
         self.assertEqual(streamed._pending_chunk, None)
 
+    def test_read_w_index(self):
+        ROW_COUNT = 2000
```

Comment:
I will say this, even though I know you won't like it. `ROW_COUNT` is not a constant or a global, it's a local variable.